src/ServiceInsight/SequenceDiagram/ModelCreator.cs

```diff
@@ -100,6 +102,17 @@ void Initialize()
             }
         }
 
+        private void OrderHandlers()
+        {
+            var start = handlers.Where(h => h.ID == ConversationStartHandlerName);
```

Comment:
this is about ordering to the best of our knowledge as the endpoints time could be out of sync. We know that logically the start of the conversation happens first, that's why it was special cased.